### ISSUE TITLE
[Tree] Forget where the user was navigated to in the tree

### ISSUE THREAD

**Issue by akhenry** [nasa/openmct#3648]
Currently the tree remembers where the user was navigated to by storing the location in localstorage (under `mct-expanded-tree-node`). This causes confusion when users load an object from a bookmark, or load a shared URL because the navigation state of the tree is inconsistent with the navigated object.

Tree navigation state is useful for remembering where you were in the tree if you have to hit refresh, but we should not treat this as a typical use case. Users should never need to refresh, and we should address the issues that lead to refresh rather than treating refresh as a common use case.

TL;DR The tree should not remember where the user is navigated to.

**Comment by akhenry** [comment]
Assigned to Joseph Lin

**Comment by johanfforsberg** [comment]
This may be related to an issue I'm seeing: if I snapshot a telemetry plot, and look at the snapshot, I'm stuck in the "fixed" timescale view. Even modifying the URL does not consistently work (though I'm not sure it's not the browser just messing it up with auto-completion somehow). In any case, to me the most intuitive thing would be that the URL overrides anything else.

A related question; if I've entered "fixed" timespan mode, what's the intended way to get out of that? The reset button on the plot appears to always go back to the snapshot time window in the case above. Modifying the URL manually does not make sense as a general UI, even if it did work. Neither does reloading the page. As far as I can see, there's only a popup that appears to tell me when I enter "fixed" timescale mode, but no indication of what mode I'm currently in, nor how to get out of it.

**Comment by johanfforsberg** [comment]
OK I see now that the timespan mode is available in the "conductor" plugin, which seems to work fine. I just hadn't loaded it since I'm trying to learn piece by piece. Is the conductor plugin considered necessary for using OpenMCT? It seems mostly useful if there's historical telemetry available..?

**Comment by jvigliotta** [comment]
Hey @johanfforsberg the Conductor plugin is not necessary to run Open MCT. It's just provides a UI to interact with the time systems and bounds.

**Comment by johanfforsberg** [comment]
Thanks, that makes sense. But in that case I think there may be an issue with being able to get out of "fixed" time mode without the conductor.

**Comment by jvigliotta** [comment]
That's a good point @johanfforsberg and seems to be an edge case we missed, we'll discuss this more on our end, but in the meantime, for that use case, you would need to have the Conductor installed to do it through the UI. You could do it through the URL, but you need to set all the parameters correctly. And that isn't really ideal.

Ex.

**Local/Realtime**
```?tc.mode=local&tc.startDelta=1800000&tc.endDelta=30000```

**Fixed**
```?tc.mode=fixedtc.startBound=1613155620369&tc.endBound=1613157450369```

We're tracking this issue here: #3684

Thanks for bringing this to our attention!

**Comment by akhenry** [comment]
Raising to critical. There is a bug in this functionality that renders the application unusable until local storage is cleared.

**Comment by jvigliotta** [comment]
Verified Fixed - Testathon: 2/25/2021

**Comment by charlesh88** [comment]
Testathon 2/25/21 verified fixed.
